internal/configuration/command_describe.go

```diff
@@ -0,0 +1,39 @@
+package configuration
+
+import (
+	"fmt"
+
+	"github.com/spf13/cobra"
+
+	pcmd "github.com/confluentinc/cli/v3/pkg/cmd"
+	"github.com/confluentinc/cli/v3/pkg/examples"
+	"github.com/confluentinc/cli/v3/pkg/output"
+)
+
+func (c *command) newDescribeCommand() *cobra.Command {
+	cmd := &cobra.Command{
+		Use:               "describe [config-field]",
+		Short:             "Describe a user-configurable field.",
+		Args:              cobra.ExactArgs(1),
+		ValidArgsFunction: pcmd.NewValidArgsFunction(c.validArgs),
+		RunE:              c.describe,
+		Example: examples.BuildExampleString(
+			examples.Example{
+				Text: `See if update checks are enabled by describing "disable_update_check".`,
+				Code: `confluent configuration describe disable_update_check`,
+			},
+		),
+	}
+
+	return cmd
+}
+
+func (c *command) describe(cmd *cobra.Command, args []string) error {
+	field := args[0]
+	if _, ok := c.jsonFieldToType[field]; !ok {
```

Comment:
Should we create a whitelist? What if we add a new field but decide we don't want to display it here?

Reply:
The maps `jsonFieldToType` and `jsonFieldToName` technically are whitelists. If a new field is added to the `config` it also needs to be added to these maps by editing the `getSettableConfigFields` function to be visible to these commands.

The alternative would be having these maps as literals that we maintain alongside the definition of the `config` struct. I honestly think this might be more maintainable and make the code a lot easier to understand than having to modify a function every time you want a field to be visible to these commands. Wdyt?